GlyssenEngine/GlyssenEngine.csproj

```diff
@@ -11,6 +16,7 @@
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
```

Comment:
Do you need this as a separate configuration, or can you map this in the solution configuration when building Debug|x64 to use Debug|AnyCPU for GlyssenEngine?

(it doesn't let me add a comment on Release|x64, so I put it here:)
The same applies to Release|x64. If you need this as a separate configuration, then you'll probably want to add `<PlatformTarget>AnyCPU</PlatformTarget` there as well.

Reply:
It's probably just leftover cruft. I originally manually changed everything from 32 to 64 explicitly, before learning that what I really wanted was AnyCPU.